src/xcode/ENA/ENA/Source/Extensions/ENAColor.swift

```diff
@@ -8,6 +8,8 @@ import UIKit
 public enum ENAColor: String, CaseIterable {
 	// MARK: - Background Colors
 	case background = "ENA Background Color"
+	case darkBackground = "ENA Dark Background Color"
```

Comment:
is this dark different from .background dark mode?

Reply:
Yes, this returns actual black for dark mode instead of the standard dark grey